src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+
+public class Main {
+    private static final int amountOfCreatedFigures = 4;
```

Comment:
it looks like a constant, so use the rule of naming constants. 
Probably FIGURES_COUNT will be better